openslides_backend/action/relations/single_relation_handler.py

```diff
@@ -179,26 +180,44 @@ def perform(self) -> RelationFieldUpdates:
                 if self.type in ("1:1", "m:1"):
                     if len(current_value) == 0:
                         rel_update["value"] = None
-                    elif len(current_value) == 1:
-                        rel_update["value"] = current_value[0]
                     else:
-                        # We added a value to the *:1 field which was already populated.
-                        # The relation handling does currently not support this kind of
-                        # relation cascading.
-                        message = (
-                            f"You can not set {fqfield} to a new value because this "
-                            "field is not empty."
-                        )
-                        raise ActionException(message)
+                        rel_update["value"] = current_value[0]
```

Comment:
This means that the length of `current_value` must alway be 1 at this point, right? Then I would like to add an `assert` here

Reply:
The assert is [here](https://github.com/OpenSlides/openslides-backend/pull/601/files#r601538005)

Reply:
Ok